doc/advanced-usage.rst

```diff
@@ -206,7 +206,7 @@ To enable agent forwarding, git has to be executed with the ``GIT_SSH_COMMAND``
 .. code-block:: bash
 
     # start the ssh server
-    ssh-mitm --remote-host github.com --scp-interface debug_traffic
+    ssh-mitm --remote-host github.com:PORT --scp-interface debug_traffic
```

Comment:
The PORT part should be removed, because github uses port 22, which is the default port